k8s/configmap.yaml

```diff
@@ -13,4 +13,13 @@ data:
   COMMUNITY: "false"  # Enable this if you want your server to show up in the community servers tab, USE WITH SERVER_PASSWORD!
   # Enable the environment variables below if you have COMMUNITY=true
   # SERVER_PASSWORD: "yourServerPassword"
-  SERVER_NAME: "Server_Name" # Note, it seems spaces are not supported, so use underlines.
\ No newline at end of file
+  SERVER_NAME: "Server_Name" # Note, it seems spaces are not supported, so use underlines.
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: palworld-settings
+data:
+  PalWorldSettings.ini: |
+    [/Script/Pal.PalGameWorldSettings]
+    OptionSettings=(Difficulty=None,DayTimeSpeedRate=1.000000,NightTimeSpeedRate=1.000000,ExpRate=1.000000,PalCaptureRate=1.000000,PalSpawnNumRate=1.000000,PalDamageRateAttack=1.000000,PalDamageRateDefense=1.000000,PlayerDamageRateAttack=1.000000,PlayerDamageRateDefense=1.000000,PlayerStomachDecreaceRate=1.000000,PlayerStaminaDecreaceRate=1.000000,PlayerAutoHPRegeneRate=1.000000,PlayerAutoHpRegeneRateInSleep=1.000000,PalStomachDecreaceRate=1.000000,PalStaminaDecreaceRate=1.000000,PalAutoHPRegeneRate=1.000000,PalAutoHpRegeneRateInSleep=1.000000,BuildObjectDamageRate=1.000000,BuildObjectDeteriorationDamageRate=1.000000,CollectionDropRate=1.000000,CollectionObjectHpRate=1.000000,CollectionObjectRespawnSpeedRate=1.000000,EnemyDropItemRate=1.000000,DeathPenalty=All,bEnablePlayerToPlayerDamage=False,bEnableFriendlyFire=False,bEnableInvaderEnemy=True,bActiveUNKO=False,bEnableAimAssistPad=True,bEnableAimAssistKeyboard=False,DropItemMaxNum=3000,DropItemMaxNum_UNKO=100,BaseCampMaxNum=128,BaseCampWorkerMaxNum=15,DropItemAliveMaxHours=1.000000,bAutoResetGuildNoOnlinePlayers=False,AutoResetGuildTimeNoOnlinePlayers=72.000000,GuildPlayerMaxNum=20,PalEggDefaultHatchingTime=72.000000,WorkSpeedRate=1.000000,bIsMultiplay=False,bIsPvP=False,bCanPickupOtherGuildDeathPenaltyDrop=False,bEnableNonLoginPenalty=True,bEnableFastTravel=True,bIsStartLocationSelectByMap=True,bExistPlayerAfterLogout=False,bEnableDefenseOtherGuildPlayer=False,CoopPlayerMaxNum=4,ServerPlayerMaxNum=32,ServerName="Default Palworld Server",ServerDescription="",AdminPassword="",ServerPassword="",PublicPort=8211,PublicIP="",RCONEnabled=False,RCONPort=25575,Region="",bUseAuth=True,BanListURL="https://api.palworldgame.com/api/banlist.txt")
```

Comment:
Any way we could format this better?

For example, this is how my ini is currently formatted

```ini
[/Script/Pal.PalGameWorldSettings]
OptionSettings=(
    Difficulty=None,
    DayTimeSpeedRate=1.000000,
    NightTimeSpeedRate=1.000000,
    ExpRate=1.000000,
    PalCaptureRate=1.000000,
    PalSpawnNumRate=1.000000,
    PalDamageRateAttack=1.000000,
    PalDamageRateDefense=1.000000,
    PlayerDamageRateAttack=1.000000,
    PlayerDamageRateDefense=1.000000,
    PlayerStomachDecreaseRate=1.000000,
    PlayerStaminaDecreaseRate=1.000000,
    PlayerAutoHPRegenRate=1.000000,
    PlayerAutoHpRegenRateInSleep=1.000000,
    PalStomachDecreaseRate=1.000000,
    PalStaminaDecreaseRate=1.000000,
    PalAutoHPRegenRate=1.000000,
    PalAutoHpRegenRateInSleep=1.000000,
    BuildObjectDamageRate=1.000000,
    BuildObjectDeteriorationDamageRate=0.000000,
    CollectionDropRate=1.000000,
    CollectionObjectHpRate=1.000000,
    CollectionObjectRespawnSpeedRate=1.000000,
    EnemyDropItemRate=1.000000,
    DeathPenalty=All,
    bEnablePlayerToPlayerDamage=False,
    bEnableFriendlyFire=False,
    bEnableInvaderEnemy=True,
    bActiveUNKO=False,
    bEnableAimAssistPad=True,
    bEnableAimAssistKeyboard=False,
    DropItemMaxNum=3000,
    DropItemMaxNum_UNKO=100,
    BaseCampMaxNum=128,
    BaseCampWorkerMaxNum=15,
    DropItemAliveMaxHours=1.000000,
    bAutoResetGuildNoOnlinePlayers=False,
    AutoResetGuildTimeNoOnlinePlayers=72.000000,
    GuildPlayerMaxNum=20,
    PalEggDefaultHatchingTime=72.000000,
    WorkSpeedRate=1.000000,
    bIsMultiplay=False,
    bIsPvP=False,
    bCanPickupOtherGuildDeathPenaltyDrop=False,
    bEnableNonLoginPenalty=True,
    bEnableFastTravel=True,
    bIsStartLocationSelectByMap=True,
    bExistPlayerAfterLogout=False,
    bEnableDefenseOtherGuildPlayer=False,
    CoopPlayerMaxNum=4,
    ServerPlayerMaxNum=32,
    ServerName="",
    ServerDescription="",
    AdminPassword="",
    ServerPassword="",
    PublicPort=8211,
    PublicIP="",
    RCONEnabled=False,
    RCONPort=25575,
    Region="",
    bUseAuth=True,
    BanListURL="https://api.palworldgame.com/api/banlist.txt"
)

Reply:
Interesting, I've tried to format it that way and it didn't work..

Reply:
Just tried it again and it seems that the settings are not in effect when its formatted that way

@Twinki14 are you sure its working for you?

Reply:
> Just tried it again and it seems that the settings are not in effect when its formatted that way
> 
> @Twinki14 are you sure its working for you?

Not completely confident, but it seems to for my local server I'm running in just a docker container.

Try using `|-` instead of `|`

Reply:
Interesting, I'll investigate when I have time tomorrow

Reply:
@Twinki14 I have now tried every possible way in yaml to format a multiline string, it seems that '>' and '>-' are the only way with an extra new line in between the header and the rest.